.rubocop.yml

```diff
@@ -83,11 +90,21 @@ Style/TrivialAccessors:
   Include:
     - "library/*/src/lib/**/*.rb"
 
-Style/VariableName:
+Naming/VariableName:
   Include:
     - "library/*/src/lib/**/*.rb"
   Exclude:
     - "library/network/src/lib/network/susefirewall.rb"
     - "library/network/src/lib/network/susefirewalld.rb"
     - "library/network/src/lib/network/susefirewall2.rb"
     - "library/network/src/lib/network/susefirewall2services.rb"
+
+# Some tests and examples mixin to global, where it is not so harmful
+# (not so sure about tests as it modify global namespace for multiple tests in paralel )
```

Comment:
We should fix this, simply by moving the `include` to a `describe` block. My own HackWeek project finds it a blocker for further checking.